resource/sv_admins.lua

```diff
@@ -18,7 +18,7 @@ end
 -- Variables & Consts
 local failedAuths = {}
 local attemptCooldown = 15000
-
+local isLan = GetConvarInt("sv_lan", 0) == 1
```

Comment:
I dont think this should be a statically declared var, since what if someone changes `sv_lan` from the cfx console? Convars can be updated without restarting txadmin.

Reply:
in that case, im pretty sure txadmin would break anyway, since it launched in sv_lan and the web side is already populated with sv_lan based information

Reply:
Its not, there are 0 references to `sv_lan` in a runtime context.

Reply:
ah okay, in which case, i will change it to get the value during the event :)